OpenRA.Mods.Common/Traits/FieldDescriptionTooltip.cs

```diff
@@ -0,0 +1,45 @@
+﻿#region Copyright & License Information
+/*
+ * Copyright 2007-2018 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.Traits
+{
+	[Desc("Shown info on the build palette widget for field units.")]
+	public class FieldDescriptionTooltipInfo : ITraitInfo
+	{
+		[Desc("Text shown in tooltip.")]
+		[Translate]
+		public readonly string Description = "";
+
+		public object Create(ActorInitializer init) { return new FieldDescriptionTooltip(this); }
+	}
+
+	public class FieldDescriptionTooltip : IProvideTooltipInfo
```

Comment:
This name doesn't sound right to me.  How about just `TooltipDescription`?

Reply:
Don't mind changing it. I just wanted to be apart of just Tooltip, since it can be seen as a genereal tooltip, and this is only shown in field units.

Reply:
The term 'Field' itself could be confusing here for some people, as Field is also a term in programming.

@pchote's suggestion sounds better.